.github/workflows/triage.yml

```diff
@@ -17,3 +22,20 @@ jobs:
       with:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
         sync-labels: true
+  notify:
+    name: Notify
+    runs-on: ubuntu-20.04
```

Comment:
Can we use 22.04 as standard?

Reply:
Or just ubuntu-latest?
I had that first, but then I assumed there was probably a reason to use 20.04 on the original workflow ... but checking now, I see that hasn't been touched for two years, so there's probably no reason after all. ;)